modeling/dataset/base.py

```diff
@@ -246,23 +247,33 @@ def create_from_config(cls, config, **kwargs):
                     config['name'], (len(train_dataset[domain]) + len(test_dataset[domain])) / max_user_idx_by_domain[domain] / max_item_idx_by_domain[domain]
             ))
 		
+        # Q: Traceback (most recent call last):
```

Comment:
For me this approach looks like a kostyil. Why is it better to transfer fields from one key to another instead of adding them in the config in the beginning?
But I'm ok if you're ok

Reply:
Maybe it is better to send something like 
`dict(config['samplers'], **{'target_domain':target_domain,'other_domains': other_domains})`
instead of changing the `config `variable